docs/_layouts/default.html

```diff
@@ -2,6 +2,19 @@
 <html lang="{{ site.lang | default: "en-US" }}">
 <!-- Site hosted on github pages, style derived from orderedlist's minimal theme -->
   <head>
+    {% if page.layout == 'post' %}
+        <title>{{ page.title }} - {{ site.title }}</title>
```

Comment:
Would you not prefer to do this in the `{% else %}` case too, i.e., basically move this outside the `if-else` and remove the `else`?

Reply:
(do you get to see this automatically @gijshendriksen or did I indeed have to tag you as I did now?)

Reply:
I did it like this to ensure the title of the homepage is just "NoGA". If we move it outside the `if-else`, this becomes "NoGA - NoGA". I can change it in a separate PR if you want, though?

(And I got a notification for the first comment as well, though that might just be my notification settings)

Reply:
Ah, I see - no that's fine as it is then!